pkg/k8s/watchers/cilium_local_redirect_policy.go

```diff
@@ -72,7 +72,7 @@ func (k *K8sWatcher) ciliumLocalRedirectPolicyInit(ciliumLRPClient *k8s.K8sCiliu
 	)
 
 	go lrpController.Run(wait.NeverStop)
-	k.k8sAPIGroups.addAPI(k8sAPIGroupCiliumClusterwideNetworkPolicyV2)
+	k.k8sAPIGroups.addAPI(k8sAPIGroupCiliumLocalRedirectPolicyV2)
```

Comment:
Q: why don't we sync on `k8sAPIGroupCiliumClusterwideNetworkPolicyV2`, and instead we sync on `k8sAPIGroupCiliumLocalRedirectPolicyV2`?

Reply:
Per the commit description, we want LRP updates to be processed in order to keep track of active local-redirect services before k8s service sync is called.